todo_app/e2e_test.py

```diff
@@ -57,8 +84,6 @@ def test_update_to_done_new_item(driver, app_with_temp_board):
      driver.get('http://localhost:5000/')
      submit_update_to_done = '/html/body/p[18]/input[2]'
      driver.find_element_by_xpath(submit_update_to_done).click()
```

Comment:
It would be nice to also make an assertion after this, to verify that the expected behaviour is actually happening when we click. For example, we could verify that a todo does now exist on the page with the option to send it back to the todo state;
```python
undo_completion_input = '//input[@type = "submit" and @value="Return to To do"]'
assert driver.find_element_by_xpath(undo_completion_input ) != None
```

Reply:
Updated,thank you